src/screens/Profile/components/ProfileFeedHeader.tsx

```diff
@@ -551,14 +554,15 @@ function DialogInner({
               </Button>
             </View>
 
-            <ReportDialog
-              control={reportDialogControl}
-              params={{
-                type: 'feedgen',
-                uri: info.uri,
-                cid: info.cid,
-              }}
-            />
+            {info.view && (
+              <ReportDialog
+                control={reportDialogControl}
+                subject={{
+                  ...info.view,
+                  $type: 'app.bsky.feed.defs#generatorView',
+                }}
+              />
+            )}
```

Comment:
Needed to pipe this through from feed info query. In this case, it should always be defined (unless the feed fails to load entirely).

It's optional in the types bc our PWI Discover feed config needs to conform to this too, and hard-coding a view here seems silly,  since you can't report Discover from PWI anyway.